sympy/stats/rv.py

```diff
@@ -1775,7 +1775,7 @@ def sample_stochastic_process(process):
     >>> from sympy import Matrix
     >>> T = Matrix([[0.5, 0.2, 0.3],[0.2, 0.5, 0.3],[0.2, 0.3, 0.5]])
     >>> Y = DiscreteMarkovChain("Y", [0, 1, 2], T)
-    >>> next(sample_stochastic_process(Y)) in Y.state_space # doctest: +SKIP
+    >>> next(sample_stochastic_process(Y)) in Y.state_space
     True
     >>> next(sample_stochastic_process(Y))  # doctest: +SKIP
```

Comment:
Can this one be unskipped, too?

Reply:
No. It produces 0,1,2 randomly.

Reply:
@eagleoflqj means there should be no changes here(the doctest should be skipped)?